### ISSUE TITLE
Disable tick limiter by default

### ISSUE THREAD

**Issue by Xandaros** [boy0001/FastAsyncWorldedit#847]
I just got complaints from out players that "all ticks [were] messed up". We installed FAWE a while ago to do some larger WorldEdit operations without crashing the server and this worked wonderfully.
What we did not expect, however, was FAWE messing with normal gameplay. Please, if you do include surprising functionality, make it opt-in.

Falling blocks either stopped falling or simply disappeared, redstone repeaters stayed mystically powered, etc. It was a mess.
After disabling this limiter, everything worked fine.

Please note: I am not suggesting to improve the limiter. (Though that might be a good idea...) It is my firm belief that the limiter is out of the scope of this project and modifies the gameplay in an unexpected way. As such, it should either be removed or be explicitly opt-in.

**Comment by boy0001** [comment]
The tick limiter is intended to catch malicious edits people make. Without it, ordinary players would be able to crash the server by using FAWE.

**Comment by Xandaros** [comment]
It's still not something you should push on people like this, especially without any mention of it in the readme or the wiki.

If people need it to protect themselves from malicious players, they can always turn it on, but something that changes the normal game should not be enabled by default.

See our situation for an example as to why: "Hey let's get FAWE instead of WorldEdit so we can make huge changes without crashing the server all the time!"
"Wait, why does nothing work any more‽"

The tick limiter is only useful for public servers anyway. There is no reason for it to exist in whitelisted servers.
(Besides, couldn't a malicious person just... like... delete everything instead? Seems a much better option to me)

**Comment by boy0001** [comment]
@Xandaros 
Interfering with

**Comment by boy0001** [comment]
@Xandaros 


> without any mention of it in the readme or the wiki.
Most people don't read the wiki. It prints a message to console whenever it does anything. I can send a message to ALL opped players if you think that would be more appropriate? 

> changes the normal game should not be enabled by default.
It's not intended to change normal gameplay. I've made some changes which should fix that. 

> The tick limiter is only useful for public servers anyway. There is no reason for it to exist in whitelisted servers.

You've never accidentally set things to the wrong id? Okay. I've changed it to default to disabled when you install the plugin on a whitelisted server. 

> couldn't a malicious person just... like... delete everything instead?

That's what the region-restrictions are for. If you don't use any protection plugin, all edits should still be logged (use `//inspect` and `//rollback`).
